pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -199,8 +207,8 @@ class RpmFirstStage(Stage):
     that should exist in the new :class:`~pulpcore.plugin.models.RepositoryVersion`.
     """
 
-    def __init__(self, remote, repository, deferred_download, optimize, skip_types=[], new_url=None,
-                 treeinfo=None):
+    def __init__(self, remote, repository, deferred_download, optimize=True, skip_types=[],
+                 new_url=None, treeinfo=None):
```

Comment:
`optimize` was declared as positional arg, which broke travis when this param was not sent